mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -175,9 +179,15 @@ class CompletionsProvider(
 
       item.setAdditionalTextEdits(additionalEdits.asJava)
 
-      completion.documentation
-        .filter(_.nonEmpty)
-        .foreach(doc => item.setDocumentation(doc.toMarkupContent))
+      val doc = completion match
+        case v: CompletionValue.Symbolic =>
+          item.setData(
+            CompletionItemData(
+              SemanticdbSymbols.symbolName(v.symbol),
+              buildTargetIdentifier
```

Comment:
@tanishiking there is a last parameter to `CompletionItemData` to say whether the completion is an override. Depending on which we merge first, we will need to add it either in this PR or in your override one.

Reply:
Ahh, I was wondering how Scala2 replaces java parameters like `x$1` with proper ones, but this is the trick https://github.com/scalameta/metals/blob/d1f2bdbd74174f266597ce7cd65497288032f73a/mtags/src/main/scala-2/scala/meta/internal/pc/CompletionItemResolver.scala#L26-L35 !
Gotcha, thank you!